pages/bookmarks.jsx

```diff
@@ -21,14 +21,22 @@ export default React.createClass({
   handleLogOutBtnClick(event) {
     event.preventDefault();
     user.logout();
-    this.setState({ user });
   },
   componentDidMount() {
     // get IDs of user's bookmarked entries
     this.setState({bookmarks: getBookmarks()});
 
-    // let's verify what the logged in status is
-    user.verify(this.props.router.location, () => this.setState({ user }));
+    user.addListener(this);
```

Comment:
pretty sure it's not, based on https://github.com/mozilla/network-pulse/blob/2ad7cee35201d9c9bad6e98bf217b2a5b3eb697d/js/app-user.js#L166 -- we create a singleton instance of the User class, and everything then gets to use that single instance as a shared object.